Library/test-helpers/lib.sh

```diff
@@ -1883,20 +1883,20 @@ limeconDeleteNetwork() {
 true <<'=cut'
 =pod
 
-=head2 limeconPrepareAgentImage
+=head2 limeconPrepareImage
 
-Prepare podman image with keylime agent. Install keylime agent and
-copy CA cert from the host into image and set needed permissions.
+Prepare podman image. Specify docker file and name tag for building images.
+Also set up ssh access for each container.
 
-    limeconPrepareAgentImage [-f|--file DOCKER_FILE] TAG
+    limeconPrepareImage [-f|--file DOCKER_FILE] [-t|--tag TAG]
```

Comment:
remove these `-f` and `-t` switches, we can work directly with positional arguments now when using them is mandatory. Your code below doesn't allow switching the options order anyway.